docs/tutorials/tutorial-jupyter-docker.md

```diff
@@ -193,9 +220,30 @@ as well as the [Python client for Druid](tutorial-jupyter-index.md#python-api-fo
 
 You should now be able to access and complete the tutorials.
 
+## Tutorial setup for ARM-based devices
+
+For ARM-based devices, follow this setup to start Druid externally, while keeping Kafka and Jupyter within the Docker Compose environment:
+
+1. Start Druid using the `start-druid` script. You can follow [Quickstart (local)](./index.md) instructions. The tutorials
+   assume that you are using the quickstart, so no authentication or authorization is expected unless explicitly mentioned.
+2. Start either Kafka and Jupyter or only Jupyter using the following commands in the same directory as `docker-compose.yaml` and `environment`:
+   
+   ```bash
+   # Start Kafka and Jupyter
+   DRUID_VERSION={{DRUIDVERSION}} docker compose --profile kafka-jupyter up -d
+
+   # Start only Jupyter
+   docker compose --profile jupyter up -d
+   ```
```

Comment:
```suggestion
   ```bash
   # Start only Jupyter
   docker compose --profile jupyter up -d
   
   # Start Kafka and Jupyter
   DRUID_VERSION={{DRUIDVERSION}} docker compose --profile kafka-jupyter up -d

   ```
```